packages/bpk-component-accordion/src/BpkAccordion.module.scss

```diff
@@ -16,12 +16,14 @@
  * limitations under the License.
  */
 
-@import '../../bpk-mixins/index.scss';
+@layer component {
+  @import '../../bpk-mixins/index.scss';
```

Comment:
![Screenshot 2024-01-05 at 16 43 12](https://github.com/Skyscanner/backpack/assets/89925955/d4c50c64-f09a-44e1-9bb5-4ce4ef9a0e8c)

Is there a reason that we need `bpk-mixins/index.scss` inside the @layer statement. This should already be wrapped in a layer higher in the stylesheet. Namely `base`.

Reply:
We can't do that because of scope but from recent discussions we have observed that mixins can be declared fine above, outside of the layers

Reply:
Correction on this for anyone viewing later, we found that we can actually import the mixins outside of the layers and we will make this change in the near future